frontend/src/sidebar/sidebar.jsx

```diff
@@ -0,0 +1,67 @@
+import './Sidebar.css';
+import {useState} from 'react';
+
+function Sidebar(props) {
+  const [isOpen, setIsOpen] = useState('');
+
+  const [isFiltersActive, setIsFiltersActive] = useState(false);
+  const [filterName, setFilterName] = useState('');
+
+  const [isSortByActive, setIsSortByActive] = useState(false);
+  const [sortByName, setSortByName] = useState('');
+
+  const toggleSidebar = () => setIsOpen(isOpen ? '' : 'hidden');
+
+  const filtersKeys = Object.keys(props.filters);
+  const sortByKeys = Object.keys(props.sortBy);
+
+  const toggleFilters = () => setIsFiltersActive(!isFiltersActive);
+  const changeFilterName = (key) => setFilterName(`: ${key}`);
+
+  const toggleSortBy = () => setIsSortByActive(!isSortByActive);
+  const changeSortByName = (key) => setSortByName(`: ${key}`);
+
+  return [
+    <div key="burger" id="burger" onClick={toggleSidebar}>☰</div>,
+    <aside key='sidebar' id='sidebar' className={isOpen}>
+      <div id='options' className={isOpen}>
+        <div className={`options ${isFiltersActive ? '' : 'hide'}`} onClick={toggleFilters}>
+          <h1 className='title'>{`Filters${filterName}`}</h1>
+          {filtersKeys.map((key) => {
+            return (
```

Comment:
Generally, we don't need to use the return keyword here. Just remove it and the curly braces too.